pkgs/development/python-modules/base58/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, buildPythonPackage, pytest, pyhamcrest }:
+
+buildPythonPackage rec {
+  pname = "base58";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "keis";
+    repo = "base58";
+    rev = "871dbe14fe8e9675ef262a5e06fbf3874562a28c";
```

Comment:
Any reason not to use `rev = "v${version}"`?

Reply:
Is it a good practice ? While checking on a few existing derivations I figured this was standard.